[Wrangler]: Update CI / CD docs with detailed API Token Instructions (#6119)
cloudflare/cloudflare-docs: content/workers/wrangler/ci-cd.md

To run Wrangler in a continuous integration/continuous deployment (CI/CD) environment, you must provide a Cloudflare API token and account ID.

## Create a Cloudflare API token

> To create a token, you need to follow the steps outlined on the [API Token](/api/get-started/create-token/) docs. 
> 
> ![API Token Templates](/workers/wrangler/static/templates.png)
> 
> - Select **Edit Cloudflare Workers** template, all templates are prefilled with a token name and permissions, you also need to modify the account and zone resources you want assigned to the token.
> 
> - Select continue to summary and review the permissions before selecting create token.
> 
> - Add the token to the `CLOUDFLARE_API_TOKEN` environment variable. (for example, `CLOUDFLARE_API_TOKEN=123 wrangler publish`)

Reviewer: ```suggestion
To create a token:

1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com).
2. Select the user icon on the top right of your dashboard > **My Profile**.
3. Select [**API Tokens**](https://dash.cloudflare.com/profile/api-tokens) > **Create Token**. 

![API Token Templates](/workers/wrangler/static/templates.png)

4. Select **Use template** next to **Edit Cloudflare Workers**. All templates are prefilled with a token name and permissions. You also need to modify the account and zone resources you want assigned to the token.
5. After editing your token, select **Continue to summary** and review the permissions before selecting create token.
6. Add the token to the `CLOUDFLARE_API_TOKEN` environment variable. (for example, `CLOUDFLARE_API_TOKEN=123 wrangler publish`)

Refer to [API Token](/api/get-started/create-token/) for more information on set up.
```

Reviewer: Step six needs information on where the env variable needs to be added.

To run Wrangler in a continuous integration/continuous deployment (CI/CD) environment, you must provide a Cloudflare API token and account ID.

## Create a Cloudflare API token

To create a token:

1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com).
2. Select the user icon on the top right of your dashboard > **My Profile**.
3. Select [**API Tokens**](https://dash.cloudflare.com/profile/api-tokens) > **Create Token**. 

![API Token Templates](/workers/wrangler/static/templates.png)

4. Select **Use template** next to **Edit Cloudflare Workers**. All templates are prefilled with a token name and permissions. You also need to modify the account and zone resources you want assigned to the token.
5. After editing your token, select **Continue to summary** and review the permissions before selecting create token.

> 6. Create a `.env` file in your project directory and set `CLOUDFLARE_API_TOKEN` as the key and your token as the value. 

Reviewer: ```suggestion
6. Create an `.env` file in your project directory and set `CLOUDFLARE_API_TOKEN` as the key and your token as the value. 
```

```bash

> CLOUDFLARE_API_TOKEN=XXXXXXXXXXXXXXXXXXX

Reviewer: ```suggestion
CLOUDFLARE_API_TOKEN=<YOUR_API_TOKEN_VALUE>
```

```

> You can also inline the value directly with the `wrangler publish` command, For example: `CLOUDFLARE_API_TOKEN=XXX wrangler publish`.

Reviewer: ```suggestion
You can also inline the value directly with the `wrangler publish` command. For example: `CLOUDFLARE_API_TOKEN=XXX wrangler publish`.
```
